gocroaring.go

```diff
@@ -133,6 +133,13 @@ func (rb *Bitmap) Contains(x uint32) bool {
 	return answer
 }
 
+// ContainsRange returns true if the integer range is contained in the bitmap
```

Comment:
This is not wrong, but it might be interesting to document what the range is, it is `[x,y)`. Furthermore, because you are using the `uint32` type, it means that you can't select the full range (since the end point is excluded).  That is why the C function takes in a wide type (`uint64_t`). There are many ways to deal with this. For example, you could define the range as the closed range `[x,y+1]` and just add a `+ 1` later.

Reply:
Fixed.